src/ButtonGroup/index.js

```diff
@@ -85,24 +70,20 @@ class ButtonGroup extends Component<Props, State> {
     return setFieldValue(name, selectedButton);
   };
 
-  handleClick = (input) => {
-    // document.getElementById(input).click();
-    input.click();
-    let updatedState = this.state.selected.slice();
+  handleClick = (inputValue, input) => {
+    const {
+      multiSelect,
+      field: { name, value },
+      form: { setFieldValue },
+    } = this.props;
 
-    if (updatedState.includes(input) && this.props.multiSelect === true) {
-      updatedState = updatedState.filter((val) => {
-        return val !== input;
-      });
-    } else if (updatedState.includes(input)) {
-      return;
-    } else if (this.props.multiSelect) {
-      updatedState.push(input);
-    } else {
-      updatedState = [input];
+    if (!multiSelect && value === inputValue) {
+      setFieldValue(name, []);
+      setTimeout(() => {
+        // this stops the onChange from firing when we change the input
+        input.checked = false;
```

Comment:
Possibly a dumb question, why would we want to stop the `onChange` from firing? I think we will want the parent to know the value was unchecked

Reply:
What happens is that it re-checks the input and then Formik and the actual hidden input become out of sync.